content/en/docs/content-structure/layout-builder/_index.md

```diff
@@ -31,38 +31,52 @@ Expandable pairs of question/answer or header/section fields.
 
 | Label | Machine Name | Type                                 | Required | Cardinality | Help text | Field Settings | Notes |
 |-------|--------------|--------------------------------------|----------|-------------|-----------|----------------|-------|
-| Title | field_title  | Text (plain)                         | yes      | 1           |
-| Body  | field_body   | Text (formatted, long, with summary) | no       | 1           |
+| Heading  | field_heading  | Text (plain)                         | yes      | 1           |  |  | H4 |
+| Body  | field_body   | Text (formatted, long, with summary) | no       | 1           |  |  | WYSIWYG |
+
+## Branch Amenities
+
+- **Machine name:** `lb_branch_amenities`
+- **Project:** [lb_branch_amenities]
+- **Designs:** [Mobile](<>) | [Desktop](<>)
+
+Display of all amenities available at a branch location.
+
+| Label          | Machine Name                     | Type                       | Required | Cardinality | Help text | Field Settings         | Notes |
+|----------------|----------------------------------|----------------------------|----------|-------------|-----------|------------------------|-------|
+| Section heading     | (inherit)     |  |  |  |  |  | H2
+| Section subheading  | (inherit)     |  |  |  |  |  | H3
+| Amenity name | Text (plain) | Entity reference (amenities taxonomy?) | yes | unlimited |  |  | Taxonomy? |
```

Comment:
Amenity name is just the term for the amenity (i.e. "Pool", "Gymnasium", et al). From what I recall, these amenities were set up to pull from a taxonomy because there is a lot of re-use across branches, so admins should be able to add an existing term to the component, or add a new term to the taxonomy. I'm still waiting on the design from Jessye, so we can discuss further after we have designs in hand.